### DISCUSSION TITLE
quasar mode add capacitor - Error Unknown command "mode"

### DISCUSSION THREAD

**Discussion by Matman94** [quasarframework/quasar#13269] [CLI - Capacitor / Cordova modes]
Hi,
I'm doing something wrong, but I don't know what ...
I made a project with: `npm init quasar` 
and then as written here: https://quasar.dev/quasar-cli-webpack/developing-capacitor-apps/preparation
I do `quasar mode add capacitor` and then I get: `Error Unknown command "mode" `.
anyone know how to solve it?

**Comment by Matman94** [comment]
Ok, You have to have newest quasar... `quasar upgrade --install`

**Comment by BharatheshJaishankar** [comment]
There is no need for adding the mode, just type:
`quasar dev/build -m capacitor -T android/ios`
Capacitor will add the source folder by itself
